packages/editor/src/client/routes/commentList.tsx

```diff
@@ -219,6 +219,7 @@ export function CommentList() {
                   {rowData?.revisions?.length ? (
                     <RichTextBlock
                       displayOnly
+                      displayOneLine
                       disabled
                       // TODO: remove this
                       onChange={console.log}
```

Comment:
Should probably remove this?

Reply:
@faebebin We added the option mainly now to use it in the comments list table, Do you suggest we move this line to another PR?

Reply:
I guess @faebebin meant that you should remove the `onChange={console.log}`  because there's also a TODO which says "remove this"

Reply:
Sorry just noticed that is existing code :)